mona_sdk/client.py

```diff
@@ -1054,6 +1057,41 @@ def create_openai_context_class(self, context_class, openai_api_type):
             else get_dict_result(True, app_server_response["response_data"], None)
         )
 
+    @Decorators.refresh_token_if_needed
+    def initiate_csv_upload_request(
+        self,
+        file_path,
+        context_class,
+        context_id_field=None,
+        export_timestamp_field=None,
+    ):
+        """
+        A wrapper function for initiate_csv_upload_request REST endpoint.
```

Comment:
We should add a note here that this function is available only in limited specific Mona environments so people won't expect to have this

Reply:
Do you think we should elaborate on which envs this is available?

Reply:
Nope. If you have it you'd know :0